chromium_src/components/omnibox/browser/omnibox_edit_model.cc

```diff
@@ -29,6 +31,15 @@ class BraveOmniboxController : public OmniboxController {
   DISALLOW_COPY_AND_ASSIGN(BraveOmniboxController);
 };
 
+#define HANDLE_BRAVE_SCHEME_URL \
+  if (url_from_text->scheme() == content::kChromeUIScheme) { \
+    GURL::Replacements replacements; \
+    replacements.SetSchemeStr(content::kBraveUIScheme); \
+    *url_from_text = url_from_text->ReplaceComponents(replacements); \
+    *text = base::UTF8ToUTF16(url_from_text->spec()); \
```

Comment:
why are you setting  `*text` here?  I think the only thing we should be changing is `url_from_text`

Reply:
Oops. forgot to delete. Done.